cmd/simulator/go.mod

```diff
@@ -15,7 +15,7 @@ replace github.com/ava-labs/subnet-evm => ../..
 
 require (
 	github.com/VictoriaMetrics/fastcache v1.10.0 // indirect
-	github.com/ava-labs/avalanchego v1.9.6 // indirect
+	github.com/ava-labs/avalanchego v1.9.7 // indirect
```

Comment:
We also need to update README (+ compatibility json).

Reply:
README seems out-of-sync for recent releases

Reply:
Updated